mopidy_spotify_web/library.py

```diff
@@ -97,6 +100,28 @@ def browse(self, uri):
         elif uri == 'spotifyweb:albums':
             return self._cache.sortedAlbums
             # return Ref directory for all albums
+        elif uri.startswith('spotifyweb:featured-playlists') or \
+             uri.startswith('spotifyweb:new-releases') or \
+             uri.startswith('spotifyweb:categories') :
+            ids = uri.split(':')
+            try:
+                sp = spotipy.Spotify(auth=self.token)
+                #sp.trace = True
+                res = sp._get('browse/' + '/'.join(ids[1:]), limit=50)
+                arr = []
+                if res.has_key('categories'):
+                    arr += [ Ref.directory(uri='spotifyweb:categories:'+cat['id']+':playlists',
+                                    name=cat['name']) for cat in res['categories']['items'] ]
+                elif res.has_key('playlists'):
+                    arr += [ Ref.playlist(uri=playlist['uri'],
+                                    name=playlist['name']) for playlist in res['playlists']['items'] ]
+                elif res.has_key('albums'):
+                    arr += [ Ref.album(uri=album['uri'],
+                                    name=album['name']) for album in res['albums']['items'] ]
+                return arr
+            except Exception as e:
```

Comment:
Please catch more specific errors than this. Likely you want to only wrap the spotipy call. Also use `logger.error(...)`